### ISSUE TITLE
[NPC] Tormented Spirit from Agamand Mills.

### ISSUE THREAD

**Issue by DanielJ79** [dalaranwow/dalaran-wow#5449]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Rare mob named Tormented Spirit (http://ru.wowhead.com/npc=1533) doesnt respawn, i have been waiting for it for 2,5 hours.

**Expected behaviour**: Tell us what should happen instead
This should respawn every 1 hour, or even faster, it is very strange that it takes so much time for this NPC to respawn

**Steps to reproduce the problem**:

1. Check if there exist this Npc in agamand mills area
2. make it to respawn every hour
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
http://ru.wowhead.com/npc=1533 . http://wowwiki.wikia.com/wiki/Tormented_Spirit , 
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
The NPC is in-game. If it's a rare mob it's not supposed to have a regular respawn timer, especially not 1h.
Closing up
